src/screens/send/components/fee-options/fee-options.tsx

```diff
@@ -70,18 +70,24 @@ export class FeeOptionsComponent extends React.Component<
             this.props.sendingToken.symbol
         );
 
-        const fees = await blockchainInstance
-            .getClient(this.props.chainId)
-            .calculateFees(
-                this.props.account.address,
-                this.props.toAddress,
-                1,
-                tokenSendingToken.contractAddress,
-                tokenSendingToken.type
-            );
+        try {
+            const fees = await blockchainInstance
+                .getClient(this.props.chainId)
+                .calculateFees(
+                    this.props.account.address,
+                    this.props.toAddress,
+                    1,
+                    tokenSendingToken.contractAddress,
+                    tokenSendingToken.type
+                );
+
+            this.setState({ feeOptions: fees });
+            this.props.onFeesChanged(fees);
+        } catch {
+            //
```

Comment:
If the estimation fails we should at lease show something to user, live advanced fees or the default fees that are set in blockchain config

Reply:
yep, default fees would be ok